cibuild.sh

```diff
@@ -72,7 +72,7 @@ setMonoDir(){
 SKIP_BOOTSTRAP=false
 
 # Paths
-THIS_SCRIPT_PATH="`dirname \"$0\"`"
+THIS_SCRIPT_PATH=`pwd`
```

Comment:
This isn't equivalent (now it's the working directory rather than this file's directory). Original was roughly `%~dp0` in DOS-land.